monkey/infection_monkey/transport/attack_telems/base_telem.py

```diff
@@ -0,0 +1,56 @@
+from enum import Enum
+from infection_monkey.config import WormConfiguration
+import requests
+import json
+from infection_monkey.control import ControlClient
+import logging
+from infection_monkey.utils import get_host_info
+
+__author__ = "VakarisZ"
+
+LOG = logging.getLogger(__name__)
+
+
+class ScanStatus(Enum):
+    # Technique wasn't scanned
+    UNSCANNED = 0
+    # Technique was attempted/scanned
+    SCANNED = 1
+    # Technique was attempted and succeeded
+    USED = 2
+
+
+class AttackTelem(object):
```

Comment:
Why is this it's own class? If it's never used on it's own.

Reply:
This class automatically includes brief data about current machine in telem if param `machine == True`. VictimHostTelem class automatically parses VictimHost type object to extract the same brief machine info. 
Even though exploiters will send VictimHost telemetry, most ATT&CK techniques should/will be done where the monkey is running, so it makes sense to send current machine in default class.